system/watchdog_timer/watchdog_timer.c

```diff
@@ -102,6 +102,21 @@ CCP_CmdRet Cmd_WDT_START_CLEAR(const CommonCmdPacket* packet)
   return CCP_make_cmd_ret_without_err_code(CCP_EXEC_SUCCESS);
 }
 
+CCP_CmdRet Cmd_WDT_SET_TIMER(const CommonCmdPacket* packet)
+{
+  uint32_t timer;
+  int ret;
+  (void)packet;
+  timer = *(uint32_t*)CCP_get_4byte_param_from_packet(packet, 0);
+  ret = WDT_set_timer((void*)&wdt_config_, timer);
```

Comment:
```suggestion
  (void)packet;
  uint32_t timer = CCP_get_param_from_packet(packet, 0, uint32_t );
  int ret = WDT_set_timer((void*)&wdt_config_, timer);
```

- `CCP_get_4byte_param_from_packet` は使用禁止（CCPのheaderファイルに使い方かいてあるよ）
- void へのキャストは，変数を使わないことの明示なので先頭にもってきてみた


って感じかしら

Reply:
あ，ごめん voidキャスト先頭はC89だとむりだ（ごめんなさい）